src/components/ImgUpload.jsx

```diff
@@ -160,6 +163,8 @@ function ImgUpload(props) {
                 disabled={uploadingPost}
               />
               <label htmlFor="file">Upload Picture</label>
+              <Camera />
```

Comment:
should we put the component under a button so user has an option to enable camera to take photo, enabling camera directly seems off, doesn't it??